t/06-open_encrypted_db.t

```diff
@@ -0,0 +1,103 @@
+# vim:set ft= ts=4 sw=4 et:
+
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+repeat_each(2);
+
+plan tests => repeat_each() * blocks() * 5;
+
+my $pwd = cwd();
+
+our $MainConfig = qq{
+    lmdb_environment_path /tmp/test5.mdb;
```

Comment:
this test relys on test 5 to run first so it might be flakky. instead, make the encrypted file a fixture under `t/fixtures` and checked it in git.